src/AdoNetCore.AseClient/AseCommand.cs

```diff
@@ -31,25 +30,17 @@ public AseCommand(AseConnection connection)
         /// the result set can continue to stream after you call <see cref="AseDataReader.Close" />. To avoid this, make sure that you call 
         /// Cancel before closing the reader or connection.</para>
         /// </remarks>
-        public void Cancel()
+        public override void Cancel()
         {
             //todo: implement
         }
 
-        /// <summary>
-        /// Creates a new instance of a <see cref="AseParameter" /> object.
-        /// </summary>
-        IDbDataParameter IDbCommand.CreateParameter()
-        {
-            return CreateParameter();
-        }
-
         /// <summary>Parameter" /> object.
         /// </summary>
         /// <remarks>
         /// The CreateParameter method is a strongly-typed version of <see cref="IDbCommand.CreateParameter" />.
         /// </remarks>
-        public AseParameter CreateParameter()
```

Comment:
We need to expose the AseXXX types where possible. Forcing a consumer to work through the interface of the base type (or cast) is ugly, and breaks compatibility with the SAP AseClient.

This is how they do it on the SqlCommand:

```
new public SqlParameter CreateParameter()
{
    return new SqlParameter();
}
override protected DbParameter CreateDbParameter()
{
    return CreateParameter();
}
```
Since we can't use the explicit interface implementation to achieve the same, now that the base class has taken that responsibility, we will have to use `new`.

Reply:
Yep, I'll go through the diff and fix instances of this